apps/hestia/src/app/cexOnRamp/layout.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import dynamic from "next/dynamic";
+import { ReactNode } from "react";
+
+const WithdrawsProvider = dynamic(
+  () =>
+    import("@orderbook/core/providers").then((mod) => mod.WithdrawsProvider),
+  { ssr: false }
+);
+
+export default function Layout({ children }: { children: ReactNode }) {
+  return <WithdrawsProvider>{children}</WithdrawsProvider>;
```

Comment:
I don't think `WithdrawProvider` is required here. Can you remove it ?

Reply:
You can rather delete this file, there would be no different layout required here. We already have a layout file in `app` folder.